test/backgroundprocessor/BackgroundBlurProcessor.test.ts

```diff
@@ -960,4 +961,36 @@ describe('BackgroundBlurProcessor', () => {
       );
     });
   });
+
+  describe('BackgroundBlurVideoFrameProcessor', () => {
+    const spec: BackgroundFilterSpec = {};
+    const options: BackgroundBlurOptions = {
+      blurStrength: BlurStrength.LOW,
+    };
```

Comment:
Scope this to each `it` block instead so the individual tests don't affect each other?

Reply:
Sure. I included them in a `beforeEach` which should re-initialized before each `it`